test/User-test.js

```diff
@@ -79,7 +86,31 @@ let user;
       dailyStepGoal: 10000,
       friends: [16, 4, 8],
     });
-    expect(fakeUser.getAvgSleepDataPerDay(sleepTestData, "sleepQuality")).to.equal(0)
-    expect(fakeUser.getAvgSleepDataPerDay(sleepTestData, "hoursSlept")).to.equal(0)
-  })
+    expect(
+      fakeUser.getAvgSleepDataPerDay(sleepTestData, "sleepQuality")
+    ).to.equal(0);
+    expect(
+      fakeUser.getAvgSleepDataPerDay(sleepTestData, "hoursSlept")
+    ).to.equal(0);
+  });
+
+  it("should return the average fluid ounces consumed per day for all time", () => {
+    expect(user.calAverageFluid(hydrationData, 1)).to.equal(63);
```

Comment:
expect(user.calAverageFluid(hydrationData, 1)).to.equal(63);

Remove the "1" in the arguments passed in, since your method in the user class doesn't have a second parameter and it doesn't need one since there is no one specific date, it is just using the entire data set.

Reply:
Thank you for catching that